### PR TITLE
Maintenance

### PR DESCRIPTION
**Pull request by dicearr**
- [x] Update with new `fluture-template` changes.
- [x] Update dependencies to close #35, close #37 and close #38.
- [x] Update examples.

### CODE REVIEW

File: test/index.js [hunk 1]
```diff
@@ -91,22 +45,24 @@ function response(spec, done) {
...
-test ('.run', () => assertResolved (run (middleware (42), 2), 42));
+test ('.run', () => equivalence (run (middleware (42), 2)) (resolve (42)));
 
-test ('.reject', () => assertRejected (evalState (null) (reject (42)), 42));
+test ('.reject', () => equivalence (evalState (null) (reject (42))) (rejectF (42)));
 
 test ('.fromComputation with a resolved future', () => {
   const actual = evalState (null) (fromComputation ((rej, res) => {
     res (42);
+    return () => null;
```
Comment (dicearr) [inline test/index.js:55]: I would say this is a breaking change. Fluture <12 wasn't complaining about missing cancellation functions.

File: .eslintrc.json [hunk 2]
```diff
@@ -2,7 +2,7 @@
   "root": true,
   "extends": ["./node_modules/sanctuary-style/eslint-es6.json"],
   "parserOptions": {"sourceType": "module"},
-  "env": {"shared-node-browser": true, "es6": true},
```
Comment (Avaq) [inline .eslintrc.json:5]: Why can't momi run in the browser?

File: index.js [hunk 3]
```diff
@@ -67,16 +67,16 @@
 //. - **[Bootstrap][example-3]** an example showing application structure.
 //. - **[Real World][example-4]** how momi is being used in real life.
 
-import Future from 'fluture';
-import {StateT} from 'monastic/index.mjs';
+import Future, {fork, reject as rejectF} from 'fluture/index.js';
```
Comment (Avaq) [inline index.js]:
```suggestion
import {Future, fork, reject as rejectF} from 'fluture/index.js';
```

File: test/index.js (hunk 1 above)
Comment (Avaq) [reply test/index.js:55]: Ah, yes. That's a breaking change.

File: .eslintrc.json (hunk 2 above)
Comment (dicearr) [reply .eslintrc.json:5]: Tricky question; I would have said that `http.createServer` seems like something that only works in the _backend_. But `http` is an external dependency, the user could provide some other sort of library that can create a "server" in the browser, like https://github.com/mafintosh/browser-server. I am slightly confused. Is that what you meant?

https://github.com/fluture-js/momi/blob/6886780c3c12473fb295fc951fecd761c094d1d9/index.js#L145-L153

Comment (Avaq) [reply .eslintrc.json:5]: > I would have said that `http.createServer` seems like something that only works in the _backend_.

You are totally right. I just forgot that we actually include http server bindings in this library. They don't necessarily need to be included though - one could use a Middleware (StateFuture) monad to build their Redux middleware, for example. Let's leave it for now.

**Comment by Avaq** [comment]
@dicearr What happened with the code coverage?

**Comment by dicearr** [comment]
I've done some back-of-the-envelope calculations and read some [docs](https://docs.codecov.io/docs/codecov-delta#usage). Relative coverage concerns only the newly added lines. So; I have added 11 lines, 3 of them are uncovered so relative coverage is `(1-3/11)*100 = 72.72%`.

The offending lines are related to the `mount` function, and the code coverage differences can be seen in [Codecov](https://codecov.io/gh/fluture-js/momi/compare/6886780c3c12473fb295fc951fecd761c094d1d9...31067674159225a4d67a097c3ee7e12ea5891fec/diff#D20-146). I don't think we need to worry about it.

**Review by Avaq — APPROVED** [review]

**Pull request merged**